subcommands/el2g/devices.go

```diff
@@ -31,3 +45,22 @@ func doDevices(cmd *cobra.Command, args []string) {
 	}
 	t.Print()
 }
+
+func doDevice(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+	deviceId := args[0]
+
+	info, err := api.El2gProductInfo(factory, deviceId)
+	subcommands.DieNotNil(err)
+	fmt.Println("Hardware Type:", info.Type)
+	fmt.Println("Hardware 12NC:", info.Nc12)
+
+	fmt.Println("Secure Objects:")
+	objects, err := api.El2gSecureObjectProvisionings(factory, deviceId)
```

Comment:
It seems strange that we need to call 2 API endpoints to get info about the single device.
Is it possible to have a single API for that?

Reply:
their API is really limited in this regard. i've hidden some of these multi-round trip APIs in our backend where its hard to execute them correctly and in the proper order. But I decided this one didn't warrant it.